test/putTransferSpec.js

```diff
@@ -1016,13 +1016,13 @@ describe('PUT /transfers/:id', function () {
       .post('/notifications', (body) => {
         const idParts = body.id.split('/')
         const notificationId = idParts[idParts.length - 1]
-        expect(body).to.deep.equal({
+        expect(_.omit(body, 'signature')).to.deep.equal({
           event: 'transfer.update',
           id: subscription.id + '/notifications/' + notificationId,
           subscription: subscription.id,
           resource: transferResult
         })
-        // TODO validate this against notification schema
+        expect(validator.validateNotification.bind(validator.validateNotification, {body: body})).to.not.throw(Error)
```

Comment:
Why omit the signature? Shouldn't it be part of a valid Notification according to the schema (https://github.com/interledger/five-bells-shared/blob/master/schemas/Notification.json)